tools/code-owners-parser/Azure.Sdk.Tools.RetrieveCodeOwners.Tests/CodeownersManualAnalysisTests.cs

```diff
@@ -42,17 +42,22 @@ public class CodeownersDiffTests
     // <commonParent>/azure-sdk-for-.../
     // ...
     private const string AzureSdkForNetTargetDirPathSuffix = "/../azure-sdk-for-net";
-    private const string AzureSdkForNetCodeownersPathSuffix = AzureSdkForNetTargetDirPathSuffix + "/.github/CODEOWNERS";
+    private const string AzureSdkForPythonTargetDirPathSuffix = "/../azure-sdk-for-python";
```

Comment:
Do we really want to keep updating this or should we just list them all?

Reply:
Yeah, in next PR I will add all the remaining repos at once.